apps/simple-trading-app/src/app/hooks/use-resize.ts

```diff
@@ -0,0 +1,36 @@
+import { useRef, useEffect, useState } from 'react';
+
+const useResize = () => {
```

Comment:
suggestion (non-blocking): Several other places also have similar functionality

- apps/token/src/components/nav/nav.tsx
- apps/trading/pages/markets/[marketId].page.tsx
- libs/market-depth/src/lib/orderbook.tsx

Might be nice to add this to react-helpers

Reply:
I have moved it into react-helpers, but haven't changed it in the individual apps, because I am currently working on another story already. I will add this to our tech debt board.